core/tauri/src/updater/core.rs

```diff
@@ -757,8 +771,8 @@ fn copy_files_and_run<R: Read + Seek>(
       Command::new("msiexec.exe")
         .arg("/i")
         .arg(found_path)
-        // quiet basic UI with prompt at the end
-        .arg("/qb+")
+        .args(msiexec_args)
+        .arg("/promptrestart")
```

Comment:
Why do we want to prompt for a system restart here? Can't think of a reason to do this in the context of installing/updating _tauri_ apps.

Reply:
when using /passive the msiexec was sometimes rebooting the system automatically. `/promptrestart` sets msiexec to prompt the user for a restart if it is needed.